package.json

```diff
@@ -1,53 +1,54 @@
 {
-  "name": "scnu-lib-fe-remake",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@douyinfe/semi-ui": "^2.8.0",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/user-event": "^13.2.1",
-    "@types/jest": "^27.0.1",
-    "@types/node": "^16.7.13",
-    "@types/react": "^17.0.20",
-    "@types/react-dom": "^17.0.9",
-    "axios": "^0.26.1",
-    "classnames": "^2.3.1",
-    "less": "^4.1.2",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-router-dom": "^6.3.0",
-    "react-scripts": "5.0.0",
-    "recoil": "^0.7.0",
-    "typescript": "^4.4.2",
-    "web-vitals": "^2.1.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "devDependencies": {
-    "node-sass": "^7.0.1",
-    "sass-loader": "^7.3.1"
-  }
+	"name": "scnu-lib-fe-remake",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@douyinfe/semi-ui": "^2.8.0",
+		"@testing-library/jest-dom": "^5.14.1",
+		"@testing-library/react": "^12.0.0",
+		"@testing-library/user-event": "^13.2.1",
+		"@types/jest": "^27.0.1",
+		"@types/node": "^16.7.13",
+		"@types/react": "^17.0.20",
+		"@types/react-dom": "^17.0.9",
+		"axios": "^0.26.1",
+		"classnames": "^2.3.1",
+		"less": "^4.1.2",
+		"react": "^18.0.0",
+		"react-dom": "^18.0.0",
+		"react-router-dom": "^6.3.0",
+		"react-scripts": "5.0.0",
+		"recoil": "^0.7.0",
+		"typescript": "^4.4.2",
+		"web-vitals": "^2.1.0"
+	},
+	"scripts": {
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"test": "react-scripts test",
+		"lint": "eslint .",
+		"lint:fix": "eslint --fix ."
+	},
+	"eslintConfig": {
+		"extends": [
+			"scnu"
+		]
+	},
+	"browserslist": {
+		"production": [
+			">0.2%",
+			"not dead",
+			"not op_mini all"
+		],
+		"development": [
+			"last 1 chrome version",
+			"last 1 firefox version",
+			"last 1 safari version"
+		]
+	},
+	"devDependencies": {
+		"eslint-config-scnu": "^1.3.1",
```

Comment:
could add documentation about our lint tool(can do it later):
1. how to config and use it?
2. what is the style/format this tool take?

Reply:
ok 🌟